src/styles/blocks/catalog.scss

```
@@ -1,13 +1,14 @@
 .catalog {
-  display: flex;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-width);
+  gap: 0 48px;

```

use row-gap for grid container instead of margin-top for each card

```suggestion
  gap: 50px 48px;

```